tests/Responses/Expected/product_list_page_by_en_gb_channel.json

```diff
@@ -27,12 +27,10 @@
             ],
             "attributes":[
                 {
-                    "code":"MUG_COLLECTION_CODE",
```

Comment:
Was it needed to remove this code from response? It breaks contract with shop api plugin.